src/main/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPlugin.kt

```diff
@@ -76,6 +76,16 @@ class NexusPublishPlugin : Plugin<Project> {
                 repository,
                 registry
             )
+            val findStagingRepository = rootProject.tasks.register<FindStagingRepository>(
```

Comment:
Btw, we need to document it somewhere, to let people know. Would you see it in README or we should just mention it in the README, but provide sample used on the wiki?